src/containers/user-list/UserList.tsx

```diff
@@ -92,7 +92,10 @@ function mapStateToProps(state: AppState, ownProps: ConnectedUserListOwnProps) {
   const { hasMore, loading, userIds } = ownProps.stateSelector(state)
   const userId = getUserId(state)
   const usersMap: { [id: number]: User } = getUsers(state, { ids: userIds })
```

Comment:
do you think it's worth adding a getNonDeactivatedUsers or getActiveUsers selector?

Reply:
The other thing is: when do we care about deactivated users? If this is almost never then the above comment holds more weight and we just use that selector that returns only active users in most parts of the app. Curious what others think.

I see we've update the makeGetSearchPlaylists (and other similar ones) selector to include the deactivated filter.

Reply:
I like this idea. I might have to look into it and double check this would have the same behavior as checking `is_deactivated` explicitly everywhere

Reply:
Hmm although there are places in the PR where I getUsers for tracks that I then want to check for `is_deactivated` that instead I'd be checking for existence, and that seems to obscure what's happening, so maybe the two selector route?

Reply:
it almost feels like maybe we should have the User type be
`type User = ActiveUser | DeactivatedUser` and force us to handle both in future UI we build

just a thot

Reply:
That makes a lot of sense if we end up stripping properties from resources that are deactivated and deleted. Might follow up with that when we start doing that.